index.d.ts

```diff
@@ -10,6 +10,13 @@ declare module '@ekreative/react-native-braintree' {
     currencyCode: string;
   }
 
+  export interface PayPalOptions {
+    clientToken: string;
+    amount: string;
+    currencyCode: string;
+    userAction?: 'commit' | '';
```

Comment:
The empty string it's strange. Could share some information about that approach?